examples/three_layer_constant_fluxes.jl

```diff
@@ -327,44 +341,59 @@ fields_to_output = merge(model.velocities, model.tracers, (e=e, ϵ=dissipation))
 
 statistics_to_output = merge(primitive_statistics, subfilter_flux_statistics, tke_budget_statistics)
 
+function init_save_some_metadata!(file, model)
+    file["parameters/coriolis_parameter"] = f
+    file["parameters/density"] = 1027.0
```

Comment:
Where is a reference density used? Better to omit it if it isn't used I think.

Reply:
Good idea!